pq-crypto/bike_r2/gf2x_mul.c

```diff
@@ -20,6 +20,9 @@
 // 3n/2 + 3n/4 + 3n/8 = 3(n/2 + n/4 + n/8) < 3n
 #  define SECURE_BUFFER_SIZE (3 * R_PADDED_SIZE)
```

Comment:
Please use `R_PADDED_QW` i.e.,
```suggestion
#  define SECURE_BUFFER_QW (3 * R_PADDED_QW)
```

Then you won't need `SECURE_BUFFER_SIZE_64_BIT`